CONST_Makefile

```diff
@@ -159,9 +159,8 @@ geoportailv3/static/build/locale/lb/geoportailv3.json: geoportailv3/locale/lb/LC
 	mkdir -p $(dir $@)
 	node tasks/compile-catalog $+ > $@
 
-geoportailv3/static/build/build.js: build.json $(OL_JS_FILES) $(NGEO_JS_FILES) $(APP_JS_FILES) .build/externs/angular-1.3.js .build/externs/angular-1.3-q.js .build/externs/angular-1.3-http-promise.js .build/externs/jquery-1.9.js .build/node_modules.timestamp
-	mkdir -p $(dir $@)
-	node tasks/build.js $< $@
+geoportailv3/static/build/build.js: .build/node_modules.timestamp .build/build.js
+	awk 'FNR==1{print ""}1' $(addprefix node_modules/, jquery/dist/jquery.min.js angular/angular.min.js angular-gettext/dist/angular-gettext.min.js angular-animate/angular-animate.min.js bootstrap/dist/js/bootstrap.min.js) .build/build.js > $@
```

Comment:
I just see that it will add a line brake between files, useful it it's missing :-)